code/modules/tgui/states.dm

```diff
@@ -72,7 +72,7 @@
 
 /mob/living/silicon/robot/shared_ui_interaction(src_object)
 	if(!cell || cell.charge <= 0 || lockcharge) // Disable UIs if the Borg is unpowered or locked.
-		return STATUS_DISABLED
+		return STATUS_CLOSE
```

Comment:
Im not sure this even fixes anything. `STATUS_DISABLED` already disables interactions with UIs, so that shouldnt even be an issue.